packages/commonwealth/server/models/discord_bot_config.ts

```diff
@@ -0,0 +1,59 @@
+import * as Sequelize from 'sequelize';
+import { Model, DataTypes } from 'sequelize';
+import { ModelStatic, ModelInstance } from './types';
+
+export type DiscordBotConfigAttributes = {
+  id: number;
+  bot_id: string;
+  chain_id: string;
+  guild_id: string;
+  snapshot_channel_id: string;
+  verification_token: string;
+  token_expiration: Date;
+  verified: boolean;
+};
+
+export type DiscordBotConfigInstance =
+  ModelInstance<DiscordBotConfigAttributes>;
+
+export type DiscordBotConfigModelStatic = ModelStatic<DiscordBotConfigInstance>;
+
+export default (
+  sequelize: Sequelize.Sequelize,
+  dataTypes: typeof DataTypes
+): DiscordBotConfigModelStatic => {
+  const DiscordBotConfig = <DiscordBotConfigModelStatic>sequelize.define(
+    'DiscordBotConfig',
+    {
+      id: {
+        type: Sequelize.INTEGER,
+        autoIncrement: true,
+        primaryKey: true,
+      }, // autogenerated
+      bot_id: { type: Sequelize.STRING, allowNull: true },
+      chain_id: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        references: { model: 'Chains', key: 'id' },
```

Comment:
Typically we add these associations at the end of the file rather than inline in the sequelize.define.

Reply:
Also, we should add association attributes objects to this object's attributes, as optional properties, for use in routes/querying.